spec/unit/application/lookup_spec.rb

```diff
@@ -78,7 +78,7 @@
   end
 
 
-  context 'when asked to explain' do
+  context 'when using a proper fixture' do
```

Comment:
Heh, chose the term 'fixture' because I found it more descriptive than 'configuration' since it implies _actual data from files on disk_.